src/test/scala/zio/interop/JavaSpec.scala

```diff
@@ -184,6 +184,7 @@ object JavaSpec {
           fiberClient  <- taskClient.fork
           resultServer <- fiberServer.join
           resultClient <- fiberClient.join
+          _            <- ZIO.effectTotal(server.close())
```

Comment:
Without this subsequent run of `test` inside sbt session causes tests to error w/o message.